src/AaveV2CollectorContractConsolidation.sol

```diff
@@ -0,0 +1,258 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.17;
+
+import {AaveV2Ethereum} from "@aave-address-book/AaveV2Ethereum.sol";
+import {AggregatorV3Interface} from "./external/AggregatorV3Interface.sol";
+import {ERC20} from "@openzeppelin/token/ERC20/ERC20.sol";
+import {SafeERC20} from "@openzeppelin/token/ERC20/utils/SafeERC20.sol";
+
+/// @title AaveV2CollectorContractConsolidation
+/// @author Llama
+/// @notice Contract to swap excess assets for USDC at a discount
+contract AaveV2CollectorContractConsolidation {
+    using SafeERC20 for ERC20;
+
+    uint256 public immutable USDC_DECIMALS;
+    uint256 public immutable ETH_USD_ORACLE_DECIMALS;
```

Comment:
Why not just have these 2 as constants instead of setting them as immutable and instantiating through the construcutor? They're static values right?